puppetfile-cli.rb

```diff
@@ -18,6 +18,7 @@ def self.parse(options)
       cache_dir: nil,
       module_paths: [],
       path: nil,
+      proxy: nil,
```

Comment:
Would it be possible to set a Forge-specific proxy? r10k permits them in configuration and Bolt also supports configuring r10k to use a Forge-specific proxy.

Reply:
`puppetfile-cli.rb` is only supposed to be an _example_   as per https://github.com/glennsarti/puppetfile-resolver/blob/master/puppetfile-cli.rb#L3-L4 , not a fully fledged CLI.

To answer your question... Yes it could.

Reply:
🤦 Ah, I completely missed that this is only providing the framework for the config and proxy support. Adding that to the resolver config is separate work then, correct?

Reply:
Nope...it's all plumbed in.  Bolt can configure a git, forge or both proxies to whatever it wants.... AND the resolver still also honors the env. vars

You'll  just need to update https://github.com/puppetlabs/bolt/blob/e3ff5ad47bd65a9081c50253b8b0a3a2b61e7e00/lib/bolt/module_installer/resolver.rb#L37-L45

``` ruby
config = PuppetfileResolver::SpecSearchers::Configuration.new.tap do |obj|
  obj.forge.proxy = ....
  obj.git.proxy = .....
end
result = resolver.resolve(
            cache:                 nil,
            ui:                    nil,
            spec_searcher_configuration: config,
            allow_missing_modules: false
          )
```

Reply:
Ooo, that's slick. Thanks!